packages/techdocs-cli/cypress/e2e/mkdocs_serve.cy.js

```diff
@@ -16,7 +16,7 @@
 /// <reference types="cypress" />
 describe('TechDocs Live Preview - MkDocs Serve', () => {
   it('successfully serves documentation', () => {
-    cy.visit(Cypress.env('mkDocsBaseUrl'));
```

Comment:
@Rugvip & @camilaibs, I can't find the cypress.json file anymore. Where these environment variables are stored?

Reply:
No idea 😅